src/modules/personenkontext/persistence/dbiam-personenkontext.repo.ts

```diff
@@ -307,4 +308,17 @@ export class DBiamPersonenkontextRepo {
 
         return groupedByPerson;
     }
+
+    public async getPersonCountByRolle(rolle: RollenArt): Promise<number> {
+        const scope: PersonenkontextScope = new PersonenkontextScope()
+            .setScopeWhereOperator(ScopeOperator.AND)
+            .findByRollen([rolle]);
+
+        const [personenkontexte]: Counted<Personenkontext<true>> = await this.findBy(scope);
+
+        const uniquePersonIds: Set<PersonID> = new Set(
+            personenkontexte.map((kontext: Personenkontext<true>) => kontext.personId),
+        );
+        return uniquePersonIds.size;
+    }
```

Comment:
I think it would be better to have a dedicated DB-query for counting, this will load every entity into memory.
This may or may not be really slow on prod, but it's not good in either case.

A `COUNT(DISTINCT personId)` query will be a lot better.

But if time is of the essence and @kristoff-kiefer approves this, I'll accept it (for now)